robot-server/tests/runs/test_run_data_manager.py

```diff
@@ -438,8 +438,9 @@ async def test_delete_current_run(
 
     await subject.delete(run_id=run_id)
 
-    decoy.verify(await mock_engine_store.clear(), times=1)
-    decoy.verify(mock_run_store.remove(run_id=run_id), times=0)
+    decoy.verify(
+        await mock_engine_store.clear(), mock_run_store.remove(run_id=run_id), times=1
```

Comment:
You should remove the `times` argument if testing call sequences. I'm [not convinced](https://mike.cousins.io/decoy/usage/verify/#verifying-a-call-count) it works...

```suggestion
        await mock_engine_store.clear(),
        mock_run_store.remove(run_id=run_id),
```